VTUL/VIVO_VT_Theme
# add email link for help

**Adds email link to header so users can request help.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1759) (:star:)

# What does this Pull Request do? (:star:)
Adds "Need help?" email link to header

# What's the changes? (:star:)
<img width="1370" alt="Screen Shot 2019-04-10 at 12 34 26 PM" src="https://user-images.githubusercontent.com/1202435/55896878-55d43e00-5b8d-11e9-8c24-27aa2d002350.png">


# How should this be tested?
* Copy files from LIBTD-1759 branch into place
* Restart tomcat
* Check that "Need Help?" link shows up in header and that it points to "researchservices@vt.edu"

# Additional Notes:
* branch: `LIBTD-1759`

# Interested parties
@yinlinchen 

(:star:) Required fields
